kmip/pie/objects.py

```diff
@@ -315,7 +331,7 @@ def __ne__(self, other):
             return NotImplemented
```

Comment:
Wow a == b does not imply b == a? That seems wrong to me, but maybe someone thought about this longer than me. OK, let's do this the Python way.